userapi/src/test/java/fk/prof/userapi/api/S3Util.java

```diff
@@ -29,9 +29,13 @@
  */
 public class S3Util {
     public void uploadFile() throws Exception {
+//        String startime = "2017-03-01T07:00:00";
+        String startime = "2017-03-13T07:00:00";
+//        FinalizedAggregationWindow window =  MockAggregationWindow.buildAggregationWindow(startime, new MockAggregationWindow.FileLoader("/Users/gaurav.ashok/Documents/fk-profiler/methodids_mock"),300, false);
+//        int durationInSeconds = 1800;
+        int durationInSeconds = 300;
 
-        String startime = "2017-03-01T07:00:00";
-        FinalizedAggregationWindow window =  MockAggregationWindow.buildAggregationWindow(startime, new MockAggregationWindow.FileLoader("/Users/gaurav.ashok/Documents/fk-profiler/methodids_mock"));
```

Comment:
Will remove the path and supply the string not from an external file but from a member variable of the S3Util class.